pywincffi/kernel32/handle.py

```diff
@@ -120,3 +120,50 @@ def WaitForSingleObject(hHandle, dwMilliseconds):
     error_check("WaitForSingleObject")
 
     return result
+
+
+def GetHandleInformation(hObject):
+    """
+    Returns properties of an object handle.
+
+    .. seealso::
+
+        https://msdn.microsoft.com/en-us/library/windows/desktop/ms724329(v=vs.85).aspx
```

Comment:
Question: both MSDN and Google searches link to the longer URLs I used. What's the source of these much nicer ones?